src/ln/channelmanager.rs

```diff
@@ -2825,8 +2825,8 @@ mod tests {
 				SecretKey::from_slice(&secp_ctx, &key_slice).unwrap()
 			};
 			let node = ChannelManager::new(node_id.clone(), 0, true, Network::Testnet, feeest.clone(), chan_monitor.clone(), chain_monitor.clone(), tx_broadcaster.clone(), Arc::clone(&logger)).unwrap();
-			let router = Router::new(PublicKey::from_secret_key(&secp_ctx, &node_id), Arc::clone(&logger));
-			nodes.push(Node { chain_monitor, tx_broadcaster, chan_monitor, node, router });
+			let router = Router::new(PublicKey::from_secret_key(&secp_ctx, &node_id), chain_monitor.clone(), Arc::clone(&logger));
+			nodes.push(Node { feeest, chain_monitor, tx_broadcaster, chan_monitor, node, router });
```

Comment:
Note that you have a rebase error here - you re-add the feest to the Node obejct (making the first commit not compile), but then you remove it again in "Add test_invalid_channel_announcemnt + test utilities".